### ISSUE TITLE
QtWebkit 5.9.1 is broken after update to Qt 5.11.x

### ISSUE THREAD

**Issue by tehnick** [mxe/mxe#2160]
diff shows:
```diff
@@ -3133,35 +3315,16 @@
 qt5/include/QtWebKit/5.9.1/QtWebKit/private/util.h
 qt5/include/QtWebKit/QtWebKit
 qt5/include/QtWebKit/QtWebKitDepends
-qt5/include/QtWebKit/QtWebKitVersion
 qt5/include/QtWebKit/qtwebkitversion.h
-qt5/include/QtWebKit/QWebDatabase
 qt5/include/QtWebKit/qwebdatabase.h
-qt5/include/QtWebKit/QWebElement
-qt5/include/QtWebKit/QWebElementCollection
 qt5/include/QtWebKit/qwebelement.h
-qt5/include/QtWebKit/QWebFullScreenVideoHandler
-qt5/include/QtWebKit/QWebHapticFeedbackPlayer
-qt5/include/QtWebKit/QWebHistory
 qt5/include/QtWebKit/qwebhistory.h
-qt5/include/QtWebKit/QWebHistoryInterface
 qt5/include/QtWebKit/qwebhistoryinterface.h
-qt5/include/QtWebKit/QWebHistoryItem
 qt5/include/QtWebKit/qwebkitglobal.h
-qt5/include/QtWebKit/QWebKitPlatformPlugin
 qt5/include/QtWebKit/qwebkitplatformplugin.h
-qt5/include/QtWebKit/QWebNotificationData
-qt5/include/QtWebKit/QWebNotificationPresenter
-qt5/include/QtWebKit/QWebPluginFactory
 qt5/include/QtWebKit/qwebpluginfactory.h
-qt5/include/QtWebKit/QWebSecurityOrigin
 qt5/include/QtWebKit/qwebsecurityorigin.h
-qt5/include/QtWebKit/QWebSelectData
-qt5/include/QtWebKit/QWebSelectMethod
-qt5/include/QtWebKit/QWebSettings
 qt5/include/QtWebKit/qwebsettings.h
-qt5/include/QtWebKit/QWebSpellChecker
-qt5/include/QtWebKit/QWebTouchModifier
 qt5/include/QtWebKitWidgets
 qt5/include/QtWebKitWidgets/5.9.1
 qt5/include/QtWebKitWidgets/5.9.1/QtWebKitWidgets
@@ -3170,21 +3333,43 @@
 qt5/include/QtWebKitWidgets/5.9.1/QtWebKitWidgets/private/qwebinspector_p.h
 qt5/include/QtWebKitWidgets/5.9.1/QtWebKitWidgets/private/qwebpage_p.h
 qt5/include/QtWebKitWidgets/5.9.1/QtWebKitWidgets/private/qwebviewaccessible_p.h
-qt5/include/QtWebKitWidgets/QGraphicsWebView
 qt5/include/QtWebKitWidgets/qgraphicswebview.h
 qt5/include/QtWebKitWidgets/QtWebKitWidgets
 qt5/include/QtWebKitWidgets/QtWebKitWidgetsDepends
-qt5/include/QtWebKitWidgets/QtWebKitWidgetsVersion
 qt5/include/QtWebKitWidgets/qtwebkitwidgetsversion.h
-qt5/include/QtWebKitWidgets/QWebFrame
 qt5/include/QtWebKitWidgets/qwebframe.h
-qt5/include/QtWebKitWidgets/QWebHitTestResult
-qt5/include/QtWebKitWidgets/QWebInspector
 qt5/include/QtWebKitWidgets/qwebinspector.h
-qt5/include/QtWebKitWidgets/QWebPage
 qt5/include/QtWebKitWidgets/qwebpage.h
-qt5/include/QtWebKitWidgets/QWebView
 qt5/include/QtWebKitWidgets/qwebview.h
```

**Comment by tehnick** [comment]
With Qt 5.10.1 it works fine.

**Comment by mabrand** [comment]
Please see https://github.com/mxe/mxe/issues/2070#issuecomment-403234353.
I think Boris was looking into updating qtwebkit.

**Comment by tehnick** [comment]
> I think Boris was looking into updating qtwebkit.

Yes, this will be a final solution, but I am not sure when it will be done. The fix of this specific issue is still needed until #2070 is not done.

**Comment by tonytheodore** [comment]
Closed by #2268
